third_party/toolchains/cpus/arm/build_raspberry_pi.sh

```diff
@@ -36,6 +36,17 @@ set -e
 
 yes '' | ./configure
```

Comment:
I think I previously missed this, but the location to this file is wrong. This folder is for toolchains bazel uses to build.
Could you move this under `tensorflow/tools/ci_build/pi` ?